From 6398e94e9ac9a170da088768198d4a2b6e989e19 Mon Sep 17 00:00:00 2001
From: Mark Wielaard <mark@klomp.org>
Date: Sun, 24 Apr 2022 17:48:39 +0200
Subject: [PATCH] elfclassify: Fix --no-stdin flag

The no-stdin option was using the wrong flag, classify_flag_stdin,
instead of classify_flag_no_stdin.

https://sourceware.org/bugzilla/show_bug.cgi?id=28724

Signed-off-by: Mark Wielaard <mark@klomp.org>
---
 src/elfclassify.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/elfclassify.c b/src/elfclassify.c
index 2f70b29a..25fe9a65 100644
--- a/src/elfclassify.c
+++ b/src/elfclassify.c
@@ -946,7 +946,7 @@ separated by newlines"), 2 },
       { "stdin0", classify_flag_stdin0, NULL, 0,
         N_("Also read file names to process from standard input, \
 separated by ASCII NUL bytes"), 2 },
-      { "no-stdin", classify_flag_stdin, NULL, 0,
+      { "no-stdin", classify_flag_no_stdin, NULL, 0,
         N_("Do not read files from standard input (default)"), 2 },
       { "compressed", 'z', NULL, 0,
 	N_("Try to open compressed files or embedded (kernel) ELF images"),
-- 
2.27.0

